syntheseus/reaction_prediction/inference/megan.py

```diff
@@ -9,7 +9,7 @@
 import os
 import sys
 from pathlib import Path
-from typing import List, Union
+from typing import Any, Dict, List, Optional, Union
```

Comment:
nit: would we not do `from __future__ import annotations` and use `dict`/`list`/etc? Or are you doing this for Pydantic?

Reply:
I can't recall what the issue with `pydantic` was anymore... Here it's mostly just old habits 🙂